data/colors_v2/vars/component_light.ts

```diff
@@ -72,7 +72,7 @@ export default {
 
     primary: {
       text: get('scale.white'),
-      bg: '#2ea44f',
+      bg: get('scale.green.4'),
```

Comment:
Should this be `get('success.emphasis')`

Reply:
Hmm.. yeah, it could. I guess it depends a bit if `green.4` should be "locked in" for buttons or change along with `success.emphasis`?

Since this seems urgent, I'll merge "as is".